### ISSUE TITLE
Issues in visualization of user types vs days

### ISSUE THREAD

**Issue by sarthak1996** [sarthak1996/Udacity-Python#7]
With filters :                
```
        PS: Case sensitivity has been disabled!
::::::::::::::::::::::Phase: Data Visualization:::::::::::::::::::::::
===============Filtering data based on following params===============
**********************************************************************
*Included Cities: Test Data/chicago , Test Data/new York City , Test Data/washington*
*                     Included Months: Mar , Jan                     *
*                   Included Days: Tue , Mon , Wed                   *
*                 Visualization type: ['User Count']                 *
*                Visualization subtype: ['Day wise']                 *
**********************************************************************
```

Received following error:
`Traceback (most recent call last):
  File "bikeshare_2.py", line 907, in <module>
    main()
  File "bikeshare_2.py", line 849, in main
    visualize_data(df.copy())
  File "bikeshare_2.py", line 764, in visualize_data
    visualize_user_counts_day_wise(df)
  File "bikeshare_2.py", line 616, in visualize_user_counts_day_wise
    if day not in df_temp.day_of_week.values:
  File "/anaconda3/lib/python3.7/site-packages/pandas/core/generic.py", line 4376, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'DataFrame' object has no attribute 'day_of_week'`

**Comment by sarthak1996** [comment]
Not fixed via [Fix for Issue #6 ](https://github.com/sarthak1996/Udacity-Python-Bikeshare/commit/fd670e33415427b53f87c71926f1e9d651589202)

**Comment by sarthak1996** [comment]
Stack trace changed after fix of Issue #6 .
Traceback (most recent call last):
  File "bikeshare_2.py", line 909, in <module>
    main()
  File "bikeshare_2.py", line 851, in main
    visualize_data(df.copy())
  File "bikeshare_2.py", line 766, in visualize_data
    visualize_user_counts_day_wise(df)
  File "bikeshare_2.py", line 618, in visualize_user_counts_day_wise
    day, None, None, i, None, None, None
  File "/anaconda3/lib/python3.7/site-packages/pandas/core/indexing.py", line 189, in __setitem__
    self._setitem_with_indexer(indexer, value)
  File "/anaconda3/lib/python3.7/site-packages/pandas/core/indexing.py", line 445, in _setitem_with_indexer
    raise ValueError("cannot set a row with "
ValueError: cannot set a row with mismatched columns

**Comment by sarthak1996** [comment]
Duplicate of #9
